crates/ef-testing/src/evm_sequencer/sequencer/mod.rs

```diff
@@ -19,43 +33,73 @@ use sequencer::{sequencer::Sequencer, state::State};
 use starknet::core::types::contract::{legacy::LegacyContractClass, CompiledClass};
 use starknet_api::{
     block::{BlockNumber, BlockTimestamp},
-    core::ChainId,
+    core::{ChainId, ClassHash, ContractAddress},
 };
 
 use super::{
-    constants::{CHAIN_ID, ETH_FEE_TOKEN_ADDRESS, STRK_FEE_TOKEN_ADDRESS, VM_RESOURCES},
+    constants::{ETH_FEE_TOKEN_ADDRESS, STRK_FEE_TOKEN_ADDRESS, VM_RESOURCES},
     types::contract_class::CasmContractClassWrapper,
     utils::compute_starknet_address,
 };
 
 /// Kakarot wrapper around a sequencer.
 #[derive(Clone)]
-pub(crate) struct KakarotSequencer(Sequencer<State, Address>);
+pub struct KakarotSequencer {
+    /// The sequencer.
```

Comment:
this comments aren't useful if we're not elaborating on their content

Reply:
I just removed it all